myuw/templates/handlebars/card/accounts/tuition.html

```diff
@@ -3,115 +3,133 @@
-			        	</div>
-			        </div>
-				</li>
-     <!-- end pce balance -->
-    {{/if}}
+			<li class="pce-tuition">
+		        <div class="card-badge clearfix" style="padding-bottom:0">
+		                <div class="pull-left">
+		                    <h4 class="card-badge-label">Amount Due<br><span style="font-size:14px;font-weight:normal;line-height:32px;">PCE-Continuum College</span></h4>
+		                </div>
+						<div class="pull-right">
+		                    <span class="card-badge-value text-danger">${{pce_accbalance}}</span>
+
+		        		</div>
+		        </div>
+		        <div class="card-badge clearfix">
+		        	<div class="pull-right"><div class="btn btn-default card-action-btn" style="margin-top:-32px;"><a href="http://portal.continuum.uw.edu" target="_blank" data-linklabel="PCE Payment portal">Make payment to PCE</a></div>
+		        	</div>
+		        </div>
+
+		</li>
+    {{else}} <!-- if PCE balance is 0 -->
+        {{#if is_c2}}
+		<li>
+            <div class="card-badge clearfix">
+                <div class="pull-left">
+                    <h4 class="card-badge-label">Amount Due<br><span style="font-size:14px;font-weight:normal;line-height:32px;">PCE-Continuum College</span></h4>
+                </div>
+                <div class="pull-right">
+                    <span class="card-badge-value">$ 0</span>
+                    <!-- <span class="card-badge-action pull-right" style="clear:both;margin-top:-6px;">Paid in full<br></span>-->
+                    <span class="card-badge-action pull-right" style="clear:both;margin-top:-6px;" ><a href="http://portal.continuum.uw.edu" target="_blank" data-linklabel="Account Statement">Account Statement</a></span>
+                </div>
+            </div>
+        </li>
+        {{/if}}
+    {{/if}}<!-- end pce balance -->
+
+    {{#if tuition_date}} <!-- N-T-S. UPDATE THE LOGIC TO REFELCT BOTH SFS + PCE -->
+		    <li class="tuition-due card-badge clearfix">
+		        <div class="pull-left">
+		            <h4 class="card-badge-label">Payment Due</h4>
+		        </div>
+				<div class="pull-right">
+		            <span class="card-badge-value" aria-hidden="true">{{toFriendlyDate tuition_due}}</span>
+					<span class="sr-only">{{toFriendlyDateVerbose tuition_due}}</span>
+		    &nbsp;
+		            <span class="card-badge-action pull-right">in {{tuition_date_offset}} days</span>
+		        </div>
+		    </li>
+    {{/if}}<!-- end tuition_date -->
 	</ul><!-- .end of the balance list -->
 
+    {{#unless is_c2_grad}} <!-- Access msg for non-PCE -->
     <div class="card-related-messages">
         <p><a href="https://sdb.admin.uw.edu/sisStudents/uwnetid/release.aspx" target="_blank">Give access to your tuition account and financial aid information</a> to parents or other third parties.</p>
+	{{/unless}}
```

Comment:
Double check the order of your closing statements here, it looks like you may have a bug. 

Above you are opening the `{{unless is_c2_grad}}`, then you open a `<div>` and `<p>` tag.  The `<p>` gets closed but the `<div>` does not before the `{{/unless}}`. This will likely cause you to have an extra `</div> in your code when the user "is_c2_grad".

Reply:
I'm also a little confused about who should see it because the comments says "non-PCE", but then the variable name is "is_c2_grad" which implies undergrad PCE students should see it. I didn't see this message when logging in as any of the undergrad PCE users.

Reply:
{{unless is_c2_grad}} moved after <div>. 
jpce , C2 undergrad displayed the message. jinter, C2 grad didn't on my build. Will double check on my-test going forward.